extensions/2.0/Khronos/KHR_node_visibility/README.md

```diff
@@ -0,0 +1,63 @@
+# KHR\_node\_visibility
+
+## Contributors
+
+- Dwight Rodgers, Adobe
+- Peter Martin, Adobe
+- Emmett Lalish, Google
+- Alexey Knyazev, Independent
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows glTF animations and interactivity to control visibility of node hierarchies. It is intended for use in conjunction with `KHR_animation_pointer` and/or `KHR_interactivity` extensions.
+
+## Extending Nodes
+
+The `KHR_node_visibility` extension object is added to the objects within the `nodes` array. The extension object contains a single boolean `visible` property. This value is mutable through JSON pointers as defined in the glTF 2.0 Asset Object Model and controls visibility of the node that contains it and all its children nodes recursively. A value of `false` causes all nodes below in the hierarchy to be omitted from display, even any nodes below that have a value of `true`.
+
+|             |   Type    |               Description              |      Required      |
+|-------------|-----------|----------------------------------------|--------------------|
+| **visible** | `boolean` | Specifies whether the node is visible. | No, default: `true`|
+
+In other words, a node is visible if and only if its own `visible` property is `true` and all its parents are visible. This allows a single change of a `visible` property at a high level of the hierarchy to hide or show complex (multi-node) objects.
```

Comment:
```suggestion
In other words, a node is visible if and only if its own `visible` property is `true` and all its ancestors are visible. This allows a single change of a `visible` property at a high level of the hierarchy to hide or show complex (multi-node) objects.
```

Reply:
The extension uses the same terms as the core spec.